roles/register_hypervisor/tasks/main.yml

```diff
@@ -28,6 +37,7 @@
                                       --host-name='{{ host_name if host_name is defined else "" }}'
                                       --host-password='{{ root_password }}'
                                       --cluster='{{ cluster_name }}'
+                                      --deploy-hosted-engine
```

Comment:
What prevents this flag from being picked up by the separate engine/hypervisors scenario? Think this role is used by both playbooks.